src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/HtmlString.cs

```diff
@@ -39,29 +37,18 @@ public static HtmlString Empty
 
         /// <summary>
         /// Writes the value in this instance of <see cref="HtmlString"/> to the target
-        /// <paramref name="targetWriter"/>.
+        /// <paramref name="writer"/>.
         /// </summary>
-        /// <param name="targetWriter">The <see cref="TextWriter"/> to write contents to.</param>
-        public void WriteTo(TextWriter targetWriter)
+        /// <param name="writer">The <see cref="TextWriter"/> to write contents to.</param>
+        /// <param name="encoder">The <see cref="IHtmlEncoder"/> with which the output must be encoded.</param>
+        public void WriteTo(TextWriter writer, IHtmlEncoder encoder)
```

Comment:
Yes. It is coming from the interface `IHtmlContent`. We currently encode the text beforehand. Hence its not being used here. I have a change coming, which delays encoding until it is written.